### PR TITLE
[js] BiDi code examples

### PR DESCRIPTION
**Pull request by TamsilAmani**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Description

Code examples for BiDi APIs

### Motivation and Context

Users can refer this to check how to use BiDi APIs in their projects

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: examples/javascript/test/bidirectional/logInspector.spec.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        })
+  
+          afterEach(async function () {
+            await driver.quit()
+          })
+
+        it('test listen to console log', async function () {
+            let logEntry = null
+            const inspector = await LogInspector(driver)
+            await inspector.onConsoleLog(function (log) {
```
Comment (harsha509) [inline examples/javascript/test/bidirectional/logInspector.spec.js]: `await inspector.onLog` i assume!

**Review by harsha509 — CHANGES_REQUESTED** [review]

File: examples/javascript/test/bidirectional/logInspector.spec.js (hunk 1 above)
Comment (TamsilAmani) [reply examples/javascript/test/bidirectional/logInspector.spec.js]: Ah! Thanks. Let me correct it.

File: examples/javascript/test/bidirectional/bidi.spec.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+const { suite } = require('selenium-webdriver/testing');
+const assert = require("assert");
+const firefox = require('selenium-webdriver/firefox');
+const LogInspector = require('selenium-webdriver/bidi/LogInspector');
```
Comment (harsha509) [inline examples/javascript/test/bidirectional/bidi.spec.js]: windows often uses a case insensitive filesystem, while linux often does not. we should do use `require('selenium-webdriver/bidi/logInspector')` instead.

**Review by harsha509 — CHANGES_REQUESTED** [review]

File: examples/javascript/test/bidirectional/bidi.spec.js (hunk 2 above)
Comment (TamsilAmani) [reply examples/javascript/test/bidirectional/bidi.spec.js]: Changes made. Thank you

**Review by harsha509 — APPROVED** [review]
Thank you @TamsilAmani !

**Pull request merged**